servicelayer/queue.py

```diff
@@ -9,17 +13,24 @@
 TASK_FINISHED = 'finished_tasks'
 TASK_TOTAL = 'total_tasks'
 INGESTION_FINISHED = 'ingestion_done'
+SLEEP = 5
+
+
+log = logging.getLogger(__name__)
 
 
 def _serialize(data):
+    data['entity'] = data['entity'].to_dict()
     return json.dumps(data)
 
 
 def _deserialize(data):
-    return json.loads(data)
+    data = json.loads(data)
+    data['entity'] = model.get_proxy(data['entity'])
```

Comment:
Do we need servicelayer to depend on followthemoney? Are there other benefits to making that link?